Add npx astro-i18next generate to the README

The readme is missing info about how to work with localized pages in dev mode.

This is what I see when issuing `npm run dev` after issuing `npx astro-i18next generate`:

<img width="1088" alt="Image" src="https://github.com/user-attachments/assets/2989f619-429e-4ca6-b5a7-55461d4941f4" />

And in the command line (WSL Ubuntu 24):
```console
WARNING(astro-i18next): "undefined" locale is not supported, add it to the locales in your astro config.
WARNING(astro-i18next): "undefined" locale is not supported, add it to the locales in your astro config.
WARNING(astro-i18next): "undefined" locale is not supported, add it to the locales in your astro config.
WARNING(astro-i18next): "undefined" locale is not supported, add it to the locales in your astro config.
WARNING(astro-i18next): "undefined" locale is not supported, add it to the locales in your astro config.
WARNING(astro-i18next): "undefined" locale is not supported, add it to the locales in your astro config.
WARNING(astro-i18next): "undefined" locale is not supported, add it to the locales in your astro config.
WARNING(astro-i18next): "undefined" locale is not supported, add it to the locales in your astro config.
WARNING(astro-i18next): "undefined" locale is not supported, add it to the locales in your astro config.
24:28:37 [200] / 204ms
24:28:37 [ERROR] Cannot read properties of undefined (reading 'map')
  Stack trace:
    at /home/johnrdorazio/development/confessit-web/node_modules/astro-i18next/src/components/HeadHrefLangs.astro:10:19
    [...] See full stack trace in the browser, or rerun with --verbose.
```

I can run `npm run preview` just fine.

--- a/package.json
+++ b/package.json
@@ -26,7 +26,7 @@
     "@types/react-dom": "^18.3.0",
     "astro": "^5.1.9",
     "astro-i18next": "^1.0.0-beta.21",
-    "i18next": "^24.2.2",
+    "i18next": "22.5.1",
     "react": "^18.3.1",
     "react-dom": "^18.3.1",
     "react-i18next": "^12.3.1",

